osf/management/commands/backfill_egap_provider_metadata.py

```diff
@@ -63,7 +63,7 @@ def backfill_egap_metadata(dry_run=False, batch_size=None):
         schema_version=Max('registered_schema__schema_version')
     ).filter(
         provider___id='egap',
-        additional_metadata=None,  # JSON fields are weird
+        additional_metadata__isnull=True,  # JSON fields are weird
```

Comment:
In https://docs.google.com/document/d/12En4sLtn5CrXZI89aNIqFjaYj26qebHC4SEYayZ_GrY/edit# `Change to `=None` filter for django.contrib.postgres.fields.JSONField`